src/wp-includes/option.php

```diff
@@ -705,9 +705,9 @@ function wp_load_core_site_options( $network_id = null ) {
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @param string      $option   Name of the option to update. Expected to not be SQL-escaped.
- * @param mixed       $value    Option value. Must be serializable if non-scalar. Expected to not be SQL-escaped.
- * @param string|bool $autoload Optional. Whether to load the option when WordPress starts up. For existing options,
+ * @param string           $option   Name of the option to update. Expected to not be SQL-escaped.
+ * @param mixed            $value    Option value. Must be serializable if non-scalar. Expected to not be SQL-escaped.
+ * @param string|bool|null  $autoload Optional. Whether to load the option when WordPress starts up. For existing options,
  *                              `$autoload` can only be updated using `update_option()` if `$value` is also changed.
  *                              Accepts 'yes'|true to enable or 'no'|false to disable.
  *                              Autoloading too many options can lead to performance problems, especially if the
```

Comment:
Same here, the documentation of all these parameter docs needs to be fixed to be consistent.